Jenkinsfile.conda

```
@@ -1,2 +1,2 @@
 @Library('JenkinsShared')_
-CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig")
+CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig", arch="noarch")
```

This syntax doesn't work the way that you expect it to. Use this instead. 
```suggestion
CondaPipeline([], "ts-ddsconfig", "lsst.ts.ddsconfig", "noarch")
```

---

OK. Fixed that.